src/importers/lastpassCsvImporter.ts

```diff
@@ -170,7 +170,28 @@ export class LastPassCsvImporter extends BaseImporter implements Importer {
                         'Number': 'number',
                         'Name on Card': 'cardholderName',
                         'Security Code': 'code',
+                        // LP provides date in a format like 'June,2020'
+                        // Store in expMonth, then parse and modify
+                        'Expiration Date': 'expMonth',
                     });
+
+                    if (mappedData[0].expMonth === ',') {
```

Comment:
Is there ever a case where `mappedData[0].expMonth` is just an empty string, or null after `parseSecureNoteMapping`?

Reply:
not in my tests. in  the `should parse blank card note` test i added, the data is from a note i exported from lastpass that literally had no fields filled out, and the result had `Expiration Date:,` as a line in the `extra` column

Reply:
Ok, but we still should guard against null reference exceptions here. Lastpass could change something in the future

Reply:
sure thing. updated